src/components/ProjectIndex/ProjectIndex.js

```diff
@@ -25,18 +28,24 @@ const ProjectIndex = (props) => {
   }
 
   return projectListLoaded === true ? (
-    <div className='main-container'>
-      <div>
-        <Button className='btn--primary' onClickHandler={onCreateProject} buttonText="Create Project" />
+    <>
+      <ProjectHeader>
+        <Button
```

Comment:
This is more of a question than a change request, but just wondering why you did it this way rather than including the button inside the `ProjectHeader` component?

Reply:
Sooo my reasoning was that the 'create project button' is specific for the projectIndex page only and if we wanted to use the projectHeader component in the future we wouldn't necessary want that button. Up for debate though / can change if wanted!

Reply:
Hmmm okay, I guess in that case we'd have to pass the text in as props as well if we wanted to reuse elsewhere?

Reply:
But maybe we can do this if/when we actually come to reuse this?

Reply:
ooo that is a _very_ good point. Shall I leave for now or refactor the button in?